account_invoice_clearing/tests/test_clearing_wizard.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from datetime import timedelta
+
+from odoo import fields
+from odoo.tests.common import tagged
+from odoo.tools import float_is_zero
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestClearingWizard(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.cl_partner = cls.env["res.partner"].create(
+            {
+                "name": "Clearing Partner",
+                "is_company": True,
+                "child_ids": [
+                    (0, 0, {"name": "Clearing Contact 1", "type": "invoice"}),
+                    (0, 0, {"name": "Clearing Contact 2", "type": "invoice"}),
+                ],
+            }
+        )
+        cls.sorted_by_seq = lambda l: sorted(l, key=lambda x: x.sequence)
+
+    def test_clearing_wizard(self):
+        def sorted_by_sequence(lines):
+            return sorted(lines, key=lambda l: l.sequence)
```

Comment:
It seems to me like both `sorted_by_seq` and `sorted_by_sequence` are the same thing. Can't we drop one?